kafka-client/src/main/java/dev/responsive/db/CassandraFactSchema.java

```diff
@@ -196,6 +200,10 @@ public void prepare(final String tableName) {
             .columns(DATA_VALUE.column())
             .where(ROW_TYPE.relation().isEqualTo(RowType.DATA_ROW.literal()))
             .where(DATA_KEY.relation().isEqualTo(bindMarker(DATA_KEY.bind())))
+            .where(TIMESTAMP.relation().isGreaterThanOrEqualTo(bindMarker(TIMESTAMP.bind())))
+            // ALLOW FILTERING is OK b/c the query only scans one partition (it actually  only
+            // returns a single value)
+            .allowFiltering()
```

Comment:
Can you elaborate on this comment -- what does `.allowFiltering()` do/mean and why would it not be "ok"?

Reply:
https://www.datastax.com/blog/allow-filtering-explained - if you don't have this you can't use `WHERE` clauses on non-clustering/primary key columns. I don't think it makes sense to elaborate too much in the comment, as this class in general assumes C* knowledge